pkg/products/threescale/reconciler_test.go

```diff
@@ -2948,6 +2964,21 @@ func TestReconciler_ping3scalePortals(t *testing.T) {
 			},
 		},
 	}
+	openshiftIngress := corev1.Service{
```

Comment:
@valerymo do we need to define the same object twice can we do without one of these.

Reply:
never mind looks like they are in different functions

Reply:
Thank you for help @austincunningham